crates/sui-protocol-config/src/lib.rs

```diff
@@ -1355,7 +1356,8 @@ impl ProtocolConfig {
     }
 
     pub fn dkg_version(&self) -> u64 {
-        self.random_beacon_dkg_version.unwrap_or(0)
+        // Version 0 was deprecated and removed, the default is 1 if not set.
+        self.random_beacon_dkg_version.unwrap_or(1)
```

Comment:
This is safe since testnet moved to version 1

Reply:
This looks unsafe to me, as it will cause validators to switch to the new version upon upgrade, mid-epoch, won't it?

Even if it is safe, I'd prefer not to do this less someone else copy this pattern later, unless there is a strong reason to do this rather instead of changing versions the usual way.

Reply:
Oh, nevermind. I see that we already switched to v1 in the config. So this won't have any effect.